src/helpers/compute-seconds.js

```diff
@@ -0,0 +1,3 @@
+const computeSeconds = (seconds) => seconds % 60;
+
+export { computeSeconds };
```

Comment:
1. `computeMinutes` and `computeSeconds` are used together, separately they don't make much sense
2. as far as I know an operation of this kind is rather known as `formatting` and not `computing`

so combing 1&2: I would go for `formatDuration`

EDIT: and `printTwoDigits` also would go inside this new function (supporting the case for `formatting`)

or look for 3rd party libraries for formatting time durations

Reply:
1. hm, `computeSeconds` is used also separately
I'm not sure I understand the whole thing, unless you mean creating one function returning the whole time format (along with a colon), not minutes and seconds separately which afterward we combine together with a colon in `render`. is this right?

I don't think there's a need for an external lib. these operations are very simple for now.

Reply:
if I checked correctly `computeMinutes` and `computeSeconds` are used exactly once, in a render method of `Timer`

you figured it out correctly, that's what I mean, a function accepting integer value corresponding to number of seconds and returning a string representation of this period (55 -> "0:55", 65 -> "1:05", 5 -> "0:05")

> I don't think there's a need for an external lib. these operations are very simple for now.

:+1:

Reply:
👍